apps/jobs/utils.py

```diff
@@ -222,3 +226,142 @@ def handle_submission_rerun(submission, updated_status):
                 "submitted_image_uri"
             ]
     return message
+
+
+def calculate_distinct_sorted_leaderboard_data(
+    user, challenge_obj, challenge_phase_split, only_public_entries
+):
+
+    # Get the leaderboard associated with the Challenge Phase Split
+    leaderboard = challenge_phase_split.leaderboard
+
+    # Get the default order by key to rank the entries on the leaderboard
+    try:
+        default_order_by = leaderboard.schema["default_order_by"]
+    except KeyError:
+        response_data = {
+            "error": "Sorry, Default filtering key not found in leaderboard schema!"
```

Comment:
nit: please change the message to `Sorry, `default_order_by` key missing in leaderboard schema!`